btree/btree.c

```diff
@@ -5985,28 +5900,24 @@ static void ut_basic_tree_oper(void)
 	temp_node = m0_alloc_aligned((1024 + sizeof(struct nd)), 10);
 	btree = m0_alloc(sizeof *btree);
 	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op, m0_btree_create(temp_node, 1024,
-							     &btree_type, nt,
-							     &b_op, seg, tx));
+							      						 &btree_type, nt, &b_op,
```

Comment:
Alignment.

Reply:
Tab size on the editor was 4 instead of 8. I will update this.